web/src/search_suggestion.ts

```diff
@@ -49,23 +49,12 @@ function channel_matches_query(channel_name: string, q: string): boolean {
     return common.phrase_match(q, channel_name);
 }
 
-function make_person_highlighter(query: string): (person: User) => string {
-    const highlight_query = typeahead_helper.make_query_highlighter(query);
-
-    return function (person: User): string {
-        return highlight_query(person.full_name);
-    };
-}
-
-function highlight_person(person: User, highlighter: (person: User) => string): UserPillItem {
-    const avatar_url = people.small_avatar_url_for_person(person);
-    const highlighted_name = highlighter(person);
-
+function user_pill_item(person: User): UserPillItem {
     return {
         id: person.user_id,
-        display_value: new Handlebars.SafeString(highlighted_name),
+        display_value: new Handlebars.SafeString(person.full_name),
```

Comment:
This is user-provided input and thus very much not safe.

Reply:
Oh sorry that was me mixing up `SafeString` and `escapeExpression`

Reply:
Oh I remember now, I saw it in `create_user_pill_context`! So it should be updated there too